Clarify LDAPS use (#113)
glpi-project/doc: source/modules/configuration/authentication/ldap.rst

* **Default server**: If you have multiple LDAP servers set up, you can only define one default server. Choosing this setting will remove it from the server it was previously set to.
* **Server** and **Port**: Represent the address and port of the LDAP directory.

>    GLPI can connect to a LDAP directory through a LDAPS connection. To enable this, prefix your server entry by *ldaps://* and change the port to the LDAPS one of your LDAP directory (default 636).

Reviewer: ```suggestion
   GLPI can connect to an LDAP directory through a LDAPS connection. To enable this, prefix your server field with *ldaps://* and change the port to the LDAPS one of your LDAP directory (default 636).
```
Minor English-related changes.
'an' should be used before a word that _sounds_ like it starts with a vowel (sounds like `el·dap`.
Prefix with sounds more natural in this case than prefix by too.

Replicates are only used when the connection to the main server is lost.
The addition of replicates in GLPI is done by entering a **Name** which is displayed in GLPI, as well as a **Server**, **Port**, and **Timeout** in the **Replicates** tab in the LDAP directory.

> To enable LDAPS, prefix your server entry by *ldaps://* and change the port to the LDAPS one of your LDAP directory (default 636).

Reviewer: ```suggestion
   GLPI can connect to an LDAP directory through a LDAPS connection. To enable this, prefix your server field with *ldaps://* and change the port to the LDAPS one of your LDAP directory (default 636).
```
